models/DataObject/ClassDefinition/Helper/DefaultValueGeneratorResolver.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Pimcore\Model\DataObject\ClassDefinition\Helper;
+
+use Pimcore\Logger;
+use Pimcore\Model\DataObject\ClassDefinition\DefaultValueGeneratorInterface;
+
+class DefaultValueGeneratorResolver
+{
+    public static $cache = [];
+
+    /**
+     * @param $generatorClass
+     *
+     * @return DefaultValueGeneratorInterface
+     */
+    public static function resolveGenerator($generatorClass)
```

Comment:
There is already Pimcore\Model\DataObject\ClassDefinition\Helper\ClassResolver::resolve
which does the same magic

Reply:
Actually this method does use `Pimcore\Model\DataObject\ClassDefinition\Helper\ClassResolver::resolve()` - what change are you thinking of?

Reply:
Comment is outdated then. Noticed that there are still changes going on. Please let me know when you are done. Thanks a lot!